src/App/AddEmployee/AddEmployeeController.java

```diff
@@ -118,5 +142,11 @@ private void insertEmployee() {
 
     @Override
     public void initialize(URL location, ResourceBundle resources) {
+        try {
+            setEmployeeId();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        box_empGender.getItems().addAll("Male","Female");
```

Comment:
Just a nitpick, I will suggest not to use both `underscore` and `camelCase` in variable names. I will suggest to use camelCase for all cases since we have followed this in entire codebase